website/tools/build_docs.js

```diff
@@ -132,32 +121,55 @@ const generateFinalMarkdown = (title, text, entityMap) => {
     return prettier.format(header + text, prettierConfig);
 };
 
+async function getExamplesFromRepo() {
+    await fs.emptyDir(EXAMPLES_DIR_NAME);
+    process.chdir(EXAMPLES_DIR_NAME);
+    const { body } = await httpRequest({
+        url: EXAMPLES_REPO,
+        json: true,
+    });
+    const builtExamples = await buildExamples(body);
+    await addExamplesToSidebars(builtExamples);
+}
+
+async function buildExamples(exampleLinks) {
+    const examples = [];
+    for (const example of exampleLinks) {
+        const responseStream = await httpRequest({
+            url: example.download_url,
+            stream: true,
+        });
+        try {
+            const pipeline = promisify(stream.pipeline);
+            await pipeline(responseStream, fs.createWriteStream(example.name));
+            const exampleName = example.name.split('.')[0];
+            examples.push(`examples/${exampleName.replace(/_/g, '-')}`);
+            console.log(`Rendering example ${example.name}`);
```

Comment:
Either put it before the action, or change to `rendered`

Reply:
Put it before the action to have the same formatting as other rendering messages.